app/src/main/res/layout-land/fragment_edit_keyboard.xml

```diff
@@ -0,0 +1,168 @@
+        app:layout_constraintBottom_toTopOf="@id/keyboard_key_holder"
+        app:layout_constraintEnd_toStartOf="@+id/save_button"
+        app:layout_constraintHorizontal_bias="0"
+        app:layout_constraintHorizontal_chainStyle="packed"
+        app:layout_constraintStart_toEndOf="@id/back_button"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <com.willowtree.vocable.customviews.VocableImageButton
+        android:id="@+id/save_button"
+        android:layout_width="@dimen/edit_keyboard_button_width"
+        android:layout_height="@dimen/edit_keyboard_button_height"
+        android:layout_marginTop="@dimen/edit_keyboard_top_margin"
+        android:layout_marginEnd="@dimen/edit_keyboard_horizontal_margin"
+        android:background="@drawable/button_default_background"
+        android:padding="@dimen/edit_keyboard_padding"
+        android:src="@drawable/ic_save_40dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <GridLayout
+        android:id="@+id/keyboard_key_holder"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_marginStart="@dimen/keyboard_side_margin"
+        android:layout_marginTop="@dimen/keyboard_keys_top_margin"
+        android:layout_marginEnd="@dimen/keyboard_side_margin"
+        android:layout_marginBottom="@dimen/edit_keyboard_bottom_margin"
+        android:columnCount="10"
+        android:rowCount="3"
+        android:useDefaultMargins="true"
+        app:layout_constraintBottom_toTopOf="@+id/keyboard_bottom_bar"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/save_button" />
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/keyboard_bottom_bar"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="@dimen/keyboard_bottom_bar_margin"
+        android:gravity="center"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="@id/keyboard_key_holder"
+        app:layout_constraintStart_toStartOf="@id/keyboard_key_holder">
+
+        <androidx.constraintlayout.widget.Guideline
+            android:id="@+id/left_guideline"
+            android:layout_width="0dp"
+            android:layout_height="match_parent"
+            android:orientation="vertical"
+            app:layout_constraintGuide_percent="0.2" />
+
+        <androidx.constraintlayout.widget.Guideline
+            android:id="@+id/right_guideline"
+            android:layout_width="0dp"
+            android:layout_height="match_parent"
+            android:orientation="vertical"
+            app:layout_constraintGuide_percent="0.8" />
+
+        <com.willowtree.vocable.customviews.VocableImageButton
+            android:id="@+id/keyboard_clear_button"
```

Comment:
Seems like a bunch of these attributes (marginEnd, background, gravity, scaleType, textSize, textStyle) are reused - worth it to extract to a style?